Superbuild/ITKExternal.cmake

```diff
@@ -89,7 +90,7 @@ if(BUILD_MOSAIC_TOOLS)
   )
 endif()
 
-SET(itk_GIT_TAG "v5.2.0")
+SET(itk_GIT_TAG "1b16b76599a5eda9f8e9665d9a3f15a87634ce92")
```

Comment:
get latest tag

Reply:
I tried a build locally with `v5.3rc01`, and it compiled fine. Good to merge, I think.